### ISSUE TITLE
Предложения на тему кланов / Suggestion about game clans

### ISSUE THREAD

**Issue by DerFieldmarshal** [romeoordos/tanktastic#45]
1. Варианты использования клановой казны. Допустим, использовать золото/серебро как ставку на клановый бой, которая при поражении передается в казну противника. Можно дать возможность главе выдавать серебро из казны на аккаунты соклановцев (ни в коем случае нельзя давать возможность передавать золото из казны на аккаунт, иначе можно создать твинк, и неосновательно обогащаться). 
2. Назначение ролей в клане. Предлагаю так:
- Лидер (все те же полномочия)
- Заместитель лидера (исключает и принимает игроков, клан распустить уже не может)
- Офицер (только принимает игроков в клан)

**Comment by DerFieldmarshal** [comment]
1. Options for using clan treasury. Suppose using gold / silver - how to bet on a clan battle. You can give the opportunity to transfer silver to the treasury on the account, if you can not get the opportunity to transfer gold to the treasury on the account, otherwise you can create a twink and unjustly enrich.
2. Assignment of roles in the clan. I suggest this:
- Leader (all the same powers)
- Deputy leaders (clan can not dissolve)
- Officer (only accept clan players)

**Comment by foxf-tastic** [comment]
> 1. Варианты использования клановой казны. Допустим, использовать золото/серебро как ставку на клановый бой, которая при поражении передается в казну противника. Можно дать возможность главе выдавать серебро из казны на аккаунты соклановцев (ни в коем случае нельзя давать возможность передавать золото из казны на аккаунт, иначе можно создать твинк, и неосновательно обогащаться).
> 2. Назначение ролей в клане. Предлагаю так:
> 
> * Лидер (все те же полномочия)
> * Заместитель лидера (исключает и принимает игроков, клан распустить уже не может)
> * Офицер (только принимает игроков в клан)

Ну золото можно и давать, но ток в ограниченных количествах, допустим, 60 монет раз в 5 дней максимум.
Званий в клане можно больше сделать:
Создатель - тот кто создал клан, имеет все права на управление им. Единственный кто может удалить клан.
Лидер - игрок назначаемый создателем аля зам. создателя, имеет все функции управления кланом, но не может его удалять.
Казначей - тот кто помимо создателя и лидера может распоряжаться казной клана.
Командир - тот кто может принимать и исключать игроков из клана.
Заместитель командира - тот кто может принимать игроков в клан, но не может их исключать.
Так же можно сделать систему званий которые не будут предоставлять фактически никаких дополнительных прав в клане, например:
Старожил - тот кто очень давно в клане и в курсе всего.
Мастер - тот кто отличается от всех остальных игроков в клане своим скиллом.
Опытный - тот кто вполне хорошо проявляет себя в боях.
Рекрут - новичок или только что вступивший в клан игрок, который себя ещё никак не зарекомендовал.

+Ко всему тобою сказанному ещё было б неплохо добавить своего рода карту, и каждому клану назначить изначально свой кусок территорий, за куски территорий кланы смогут воевать друг с другом и назначать друг другу КБ внутри игры. Это своего рода глобалка, тут идей с ней можно тонны придумать.

**Comment by hunter1486** [comment]
Да, и чтобы звания в клане отображались цветом вместе с тэгом клана(который будет перед ником) над танком

**Comment by hunter1486** [comment]
А еще можно поделить сами кланы на какие нить суперкрутые и начинающие. И даватьтм звёзды, например максимальный ранг 1 большая звезда, минимальный 5 но маленьких, ну крч идея ясна
